### ISSUE TITLE
Joystick input causing socket errors in Linux Desktop version

### ISSUE THREAD

**Issue by gleparc** [rolandoislas/drc-sim-client#9]
Hi Roland,

I am getting this error regarding my joystick input. I am using Qjoypad and openJDK on my virtual machine ubuntu linux box with the drc-sim-client. Should I be using another joystick input program/driver?

I have tried two separate gamepads, but get a similar error

![image](https://cloud.githubusercontent.com/assets/5346145/26425890/c044e696-40d6-11e7-83bd-490c4e170bb6.png)

![image](https://cloud.githubusercontent.com/assets/5346145/26424929/4949a886-40d3-11e7-8a91-b879d0393928.png)
![image](https://cloud.githubusercontent.com/assets/5346145/26425576/8db8db8e-40d5-11e7-8271-cc9486a8a1ca.png)

**Comment by opencma** [comment]
Is the server version 1.6.2? There are a few of oversights:
- I forgot to update the version number.
- The video stream config option is not logged.
- The server does not check for dead client sockets if the video stream is disabled!

If the client is a version prior to the 1.2 release it will disconnect if there is no video stream.

Were there issues with using a controller directly?

Edit: Does this only happen when using a controller?

**Comment by gleparc** [comment]
Thanks a lot for your reply. It is only server version 1.6 (I think, could not find another) - but I neglected to tell you that I was using client 1.2-beta (didn't notice you had updated the client! my fault!). Now I can report to you that there are no more problems when I used server version 1.6 and client version 1.2. The beta was the one that was disconnecting. Now the connection stays for a long time.

Btw - I can confirm that the blow (mic) button works - I just tried it with Captain Toad game and it's brilliant! You have made a great piece of software!
